.github/actions/build-setup-ubuntu/action.yml

```diff
@@ -22,48 +22,27 @@ runs:
         # to save time.
         large-packages: false
 
-    # Cache and install a recent version of LLVM. This uses the GitHub action
-    # cache to avoid directly downloading on each iteration and improve
-    # reliability.
-    - name: Cache LLVM and Clang installation
-      id: cache-llvm-ubuntu
-      uses: actions/cache@1bd1e32a3bdc45362d1e726936510720a7c30a57 # v4.2.0
-      env:
-        cache-name: cache-llvm
-      with:
-        path: ~/llvm
-        key: LLVM-16-Cache-ubuntu-${{ runner.arch }}
-
-    - name: Download LLVM and Clang installation
-      if: steps.cache-llvm-ubuntu.outputs.cache-hit != 'true'
+    - name: Show system LLVM and Clang installation
       shell: bash
       run: |
-        cd ~
-        LLVM_RELEASE=clang+llvm-16.0.4-x86_64-linux-gnu-ubuntu-22.04
```

Comment:
FWIW, could we just use the old code and use LLVM-19 releases here? I'm worried we're going to re-discover the problems we hit with the debian packaging before.

Reply:
Not unless we build our own packages. There's no ubuntu packages in the community-driven binary releases for x64 since at least 18. This is mentioned in the PR description too.

Reply:
Also, just fwiw I used these apt.llvm.org packages on subspace CI for a year without any issues, which is why I knew to suggest it.